tests/test_smartJSON.py

```diff
@@ -111,3 +111,17 @@ def test_get_attrs_by_parent_w_simple_string(nested_json_one):
                     {'location': '5 Park Av, NY, NY 10010',
                         'email': 'alice@efgcompany.com',
                         'phone': '7564525797'}]}
+
+def test_empty_string():
+    empty_string = dict(smartJSON('""'))
+    assert  isinstance(empty_string, dict)
```

Comment:
It's not possible to extract the empty values for the conversion to dict at the moment as it appears we're returning an empty list from the tree when there is no value for a node. Might need to store more data in the tree to get a perfect translation back to dict (and indeed to reproduce the original json string if needed)

Reply:
This should raise a `json.decoder.JSONDecodeError` imo as it's not really valid JSON.